packages/arb-avm-cpp/data_storage/src/value/code.cpp

```diff
@@ -34,8 +34,13 @@ namespace {
 constexpr auto max_code_segment_key = "max_code_segment";
 constexpr auto segment_key_prefix = std::array<char, 1>{87};
 constexpr auto segment_key_size = segment_key_prefix.size() + sizeof(uint64_t);
+constexpr auto metadata_value_size =
+    sizeof(uint32_t) + sizeof(uint64_t) + sizeof(uint64_t) + sizeof(uint64_t);
+constexpr auto segment_chunk_key_size =
+    segment_key_prefix.size() + sizeof(uint64_t) + sizeof(uint64_t) + 1;
```

Comment:
This doesn't need to be in this PR, but we should consider putting these in their own tables.

Reply:
Agreed it's somewhat awkward as is. My thinking is that we likely get some advantage by having the keys that will be read or written to nearby having a shared prefix so that rocksdb will hopefully be able to do the lookups more efficiently